app/controllers/v1/tasks_controller.rb

```diff
@@ -0,0 +1,41 @@
+module V1
+  class TasksController < ApplicationController
+    def index
+      @project = Project.find_by(id: params[:project_id])
+      if @project.nil?
+        head(404)
+      else
+        render json: @project.tasks
```

Comment:
The projects index allows paging of the results. It would probably be nice to support a consistent interface and allow paging of the tasks index as well.